website/huaweicloud.erb

```diff
@@ -508,7 +508,7 @@
               <a href="#">Resources</a>
               <ul class="nav nav-auto-expand">
                 <li>
-                  <a href="/docs/providers/huaweicloud/r/blockstorage_volume_v2.html">huaweicloud_blockstorage_volume_v2</a>
+                  <a href="/docs/providers/huaweicloud/r/evs_volume_v3.html">huaweicloud_evs_volume_v3</a>
                 </li>
```

Comment:
we should add a new item of the new resource rather than replace the *huaweicloud_blockstorage_volume_v2* resource.